Jenkinsfile

```diff
@@ -1,17 +1,16 @@
-
 def config = {
     owner = 'ksql'
     slackChannel = '#ksql-alerts'
     ksql_db_version = "0.7.0-SNAPSHOT"
-    cp_version = "5.5.0-beta200204173843"
-    packaging_build_number = "1"
+    cp_version = "5.5.x"
```

Comment:
We should only be using beta artifacts of confluent platform.

Reply:
Right – this was a temporary test while we're waiting on new beta artifacts. I will revert the change.

Reply:
@elismaga, we're back to the original set of changes now.

In retrospect, I should have done that test in a separate test PR. For what it's worth, the PR build passed against the other artifacts.